src/apps/api/serializers/competitions.py

```diff
@@ -101,39 +94,61 @@ class Meta:
             }
         }
 
+    def get__object_type(self, obj):
+        return "competition"
+
     def validate_description(self, description):
         if description:
             description = description.replace("<p>", "").replace("</p>", "")
         return description
 
-    def validate_producer(self, producer):
-        context_producer = self.context.get(producer)
-        if context_producer:
-            return context_producer
 
-        if not producer:
-            raise ValidationError("Producer not found when creating data entry")
-        return producer
+class CompetitionParticipantCreationSerializer(serializers.ModelSerializer):
+
+    competition = serializers.IntegerField()
```

Comment:
We can take a look at this again, but Eric and I wrestled with CompetitionParticipant serializer quite a bit. This particular serializer is so we can submit directly from v2/etc with remote_id instead of Chahub's ID/PK for the objects.

Reply:
Oh, right, because we aren't looking up on a primary key, hmm, could maybe be using `ForeignKey(to_field=...)` but that is probably more trouble that this is worth.